app/views/users/new.html.erb

```diff
@@ -1,3 +1,23 @@
-<% provide :title, t(".Sign up") %>
+<% provide :title, t(".sign_up") %>
 <h1><%= t".sign_up" %></h1>
-<p><%= t".p_signup" %></p>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for @user do |f| %>
```

Comment:
Biến f đại diện cho điều gì nhỉ? Phân biệt form for, form with, form tag

Reply:
dạ biến f là đại diện : f (for “form”).

Reply:
chưa trả lời ý thứ 2 :v